docs/api-docs/catalog/products-overview.md

```diff
@@ -1030,6 +1026,32 @@ X-Auth-Token: {{ACCESS_TOKEN}}
 }
 ```
 
+## Product Sort Order (Beta)
+
+[Product Sort Order](add-link-here) allows you to manage the sort order of products displayed on any given category page. Products assigned to multiple storefront categories can have different sort order values per category.
+
+### Product sorting on a storefront 
+
+The Catalog API supports two manually managed methods of product sorting: on a category level and on a product level. If a user combines both sorting methods on a storefront, products are sorted based on the following sorting method priority:

```

Comment:
The Catalog API supports two manually managed methods of product sorting: on a category level and a product level. If a user combines both sorting methods on a storefront, the API manually sorts the category products first and then the product-level products.

Reply:
Feel free to rewrite my suggestion.  Would you say "product-level products"?